api/interfaces/common/FormInputInterfaces.ts

```diff
@@ -0,0 +1,360 @@
+/*
```

Comment:
We have rest api's that return input descriptors defined in the vssf sdk. 
So we had to bring in this interface as the generated client refers to it.